assets/javascripts/wizard/templates/components/wizard-field.hbs

```diff
@@ -16,6 +16,12 @@
   </div>
 {{/if}}
 
+{{#if field.char_counter}}
```

Comment:
@fzngagan This should be inside the input aread so that the counter can sit in-line with any input controls, like the toggle preview button.